src/frontends/paddle/src/op/roll.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs roll(const NodeContext& node) {
+    auto input_node = node.get_input("X");
+    Output<Node> shifts_node;
+    if (node.has_input("ShiftsTensor")) {
```

Comment:
Can you also add a testcase with "ShiftsTensor" input?

Reply:
hi @meiyang-intel I think this testcase will be triggered when shift is variable as `roll_test_2`:
https://github.com/PaddlePaddle/Paddle/blob/release/2.6/python/paddle/tensor/manipulation.py#L1890

Reply:
The code show that it will be triggered when shifts is variable. But in this testcase shifts=1